packages/lab/stories/divider/divider.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import { StoryFn } from "@storybook/react";
+import { Text, StackLayout } from "@salt-ds/core";
+
+import { Divider, DividerProps } from "@salt-ds/lab";
+
+export default {
+  title: "Lab/Divider",
+  component: Divider,
+};
+
+export const Primary: StoryFn<DividerProps> = () => {
```

Comment:
Can we update these to match the site examples please

Reply:
done, updated